.circleci/config.yml

```diff
@@ -140,6 +143,35 @@ jobs:
       - store_artifacts:
           path: build/app/outputs/flutter-apk/
 
+  frontend-android-floss-build:
```

Comment:
Can't we just make the `frontend-android-build` more generic such that we pass the option `FLOSS` and don't have to define it twice? It is mostly duplicated